rpcs3/Emu/Cell/Modules/cellMouse.cpp

```diff
@@ -134,22 +134,27 @@ s32 cellMouseGetData(u32 port_no, vm::ptr<CellMouseData> data)
 
 s32 cellMouseGetDataList(u32 port_no, vm::ptr<CellMouseDataList> data)
 {
-	UNIMPLEMENTED_FUNC(sys_io);
-	
-	return CELL_OK;
+	sys_io.todo("cellMouseGetDataList(port_no=%d, data=0x%x", port_no, data);
+	if (g_cfg.io.mouse == mouse_handler::null)
+		return CELL_MOUSE_ERROR_NO_DEVICE;
+	else
+	{
+		sys_io.error("cellMouseGetDataList was called while mouse handler is not set to null, this is gonna hurt...");
```

Comment:
If data->list_num is set to 0 before returning this should hurt much less.

Reply:
Confirmed. Setting list_num to 0 fixes dark souls 2 as well

Reply:
Good idea. Fixed.